templates/element/routes_panel.php

```diff
@@ -3,13 +3,43 @@
  * @var \Cake\Routing\Route\Route[] $routes
  * @var string $matchedRoute
  */
+
+use Cake\Core\Plugin as CorePlugin;
 use Cake\Utility\Hash;
 
 $routes = Cake\Routing\Router::routes();
+
+$amountOfRoutesPerGroup = [];
+foreach ($routes as $route) {
+    $group = $route->defaults['plugin'] ?? 'app';
+    if (!array_key_exists($group, $amountOfRoutesPerGroup)) {
+        $amountOfRoutesPerGroup[$group] = 0;
+    }
+    $amountOfRoutesPerGroup[$group]++;
+}
+
+$pluginNames = [];
+foreach (CorePlugin::loaded() as $pluginName) {
+    if (!empty($amountOfRoutesPerGroup[$pluginName])) {
+        $name = sprintf('%s (%s)', $pluginName, $amountOfRoutesPerGroup[$pluginName]);
+        $pluginNames[$name] = preg_replace('/\W+/', '', strtolower($pluginName));
```

Comment:
This could be a call to `Inflector` I think.

Reply:
I am not able to find a suitable function in the Inflector Utility Class to handle a plugin name like `ADmad/Sequence`

- `Inflector::variable('ADmad/Sequence');` results in `aDmad/sequence` which is not a valid CSS class
- `Inflector::dasherize('ADmad/Sequence');` results in `a-dmad/sequence` which is also not a valid CSS class

Thats why I used this rather simple regex

Reply:
I would love to have a static cakephp function to just put any string into it and it returns me a valid CSS class.
Currently I have used a custom Helper in my app for that since it is mainly used in the template.

Reply:
Did you try Text::slug()?

Reply:
this looks good, will push that

Reply:
To be fair I would have never thought that a `Text::slug()` method could be used to convert a string to a valid CSS class.